pymc/tests/test_sampling_jax.py

```diff
@@ -61,6 +62,24 @@ def test_deterministic_samples():
     assert np.allclose(trace.posterior["b"].values, trace.posterior["a"].values / 2)
 
 
+def test_get_log_log_likelihood():
+    obs = np.random.normal(10, 2, size=100)
+    obs_at = aesara.shared(obs, borrow=True, name="obs")
+    with pm.Model() as model:
+        a = pm.Normal("a", 0, 2)
+        sigma = pm.HalfNormal("sigma")
+        b = pm.Normal("b", a, sigma=sigma, observed=obs_at)
+
+        trace = pm.sample(chains=1, random_seed=1322)
```

Comment:
We can run a smaller sampling with multiple chains like `(tune=10, draws=10, chains=2)`. That should be as good of a test but also covers multiple chains

Reply:
Sounds good.